### DISCUSSION TITLE
New Discord Server

### DISCUSSION THREAD

**Discussion by jzeiber** [aduros/wasm4#187] [General]
Somehow I lost the retro game console discord server, and the Discord link on the wasm4 site is expired.  Can you refresh the invite link?  Thanks.

**Comment by FaberVitale** [comment]
I've got the same issue.

**Comment by aduros** [comment]
Hey all, bad news... the entire Fantasy Console Discord server was deleted afer the server admin's account was compromised :cry: 

I haven't heard any word about when (or even if) the server can be restored, so I went ahead and set up a new WASM-4 Discord. Please join using this invite link: https://discord.gg/7teRdHdbYk

Creating our own Discord was something I planned to do anyways at some point, but I feel really bad for TIC-80 and all the other communities on that server.

**Reply by SergioFLS** [reply]
Just suggesting, but have you seen [Matrix](https://matrix.org)? It's an open message protocol that supports connecting to other self-hosted instances, along with "bridges" that would connect with something like a Discord channel.

On a server I own I started bridging to Matrix as I didn't like much of Discord having anti-privacy requirements and other bad stuff on the ToS, and feared of forcing people to follow most of such bad stuff.

As of bridging, there's a bot called [matrix-appservice-discord](https://github.com/Half-Shot/matrix-appservice-discord) that you could either self-host with a self-hosted instance, or use the public instance at [https://t2bot.io](https://t2bot.io).

Sorry if I'm being spammy, I just wanted to share my thoughts :/

**Reply by vintprox** [reply]
I appreciate an idea from @SergioFLS. Seeing as how TIC-80 is very reliant on Discord, while there might be a room for open platform, I'm a bit disappointed that fantasy computer communities like WASM-4 settle with proprietary software, by extension.
